tests/integration/components/o-s-s/attribute-text-test.ts

```diff
@@ -0,0 +1,53 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+import click from '@ember/test-helpers/dom/click';
+
+module('Integration | Component | o-s-s/attribute-text', function (hooks) {
+  setupRenderingTest(hooks);
+
+  module('Default behavior', function () {
+    test('it renders', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" @value="World" />`);
+      assert.dom('.oss-attribute--text').exists();
+    });
+
+    test('it renders without a value', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" />`);
+      assert.dom('.oss-attribute--text').exists();
+    });
+
+    test('it renders with a tooltip when specified', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" @tooltip="Tooltip" />`);
+      assert.dom('.oss-attribute--info').exists();
+    });
+
+    test('it renders a dash as value when no value is specified', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" />`);
+      assert.dom('.oss-attribute--text').exists();
+      assert.dom('.oss-attribute--text--value').hasText('-');
+    });
+
+    test('it copies the value', async function (assert) {
+      const writeTextStub = sinon.stub(navigator.clipboard, 'writeText').resolves();
```

Comment:
Don't forget to call `restore()` on `writeTextStub` at the end of the test, otherwise you will break other tests (it is the case here)